index.html

```diff
@@ -3,139 +3,135 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+          integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
     <script src="https://kit.fontawesome.com/aa7a31f950.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
     <style>
-    .wrapper {
-        min-height: calc(100vh - 15px);
-        margin: 15px auto 0;
-    }
+        .wrapper {
+            min-height: calc(100vh - 15px);
+            margin: 15px auto 0;
+        }
     </style>
-    <title>LogicPart</title>
+    <title>Task Tracker</title>
```

Comment:
add script with theme in head before all other scripts if you have any

Reply:
I tried to add script with theme changing in <head>, but took error, that I cant refer to "body".
So I decided to add transition only when user changes theme.
```suggestion
 document.addEventListener("DOMContentLoaded", () => {
    if (JSON.parse(localStorage.getItem(LocalStorageKeys.darkMode))) {
        body.classList.add(THEME_CLASSNAMES.dark);
    }
});
toggleTheme.addEventListener('click', () => {
    body.classList.add(THEME_CLASSNAMES.transition);
    body.classList.toggle(THEME_CLASSNAMES.dark);
    updateLocalStorage(LocalStorageKeys.darkMode, body.classList.contains(THEME_CLASSNAMES.dark));
});
```

Reply:
It's possible to add it directly after open `body` tag too to solve this problem.